packages/ripple-nuxt-ui/lib/module.js

```diff
@@ -13,6 +12,17 @@ module.exports = function nuxtRipple (moduleOptions) {
     options: options
   })
 
+  if (!this.options.styleResources) {
+    this.options.styleResources = {
+      scss: [
+        // Theme - Injects sass variables into components
+        resolve(__dirname, `${appDir}/assets/_theme.scss`)
```

Comment:
Should we check that _theme.scss exists before resolving and enabling the module?